tests/TestDateTime.elm

```diff
@@ -282,14 +282,14 @@ toFromISO8601 =
                         |> setMillisecond 6
                         |> toISO8601
                         |> Expect.equal "1970-01-01T00:00:00.006Z"
-            , fuzz2 (intRange -23 23) (intRange 0 59) "fromISO8601 parses offsets correctly" <|
-                \hour minute ->
+            , fuzz4 (intRange -23 23) (intRange 0 59) (oneOf [ constant "-", constant "−" ]) (oneOf [ constant ":", constant "" ]) "fromISO8601 parses offsets correctly" <|
+                \hour minute negStr separator ->
                     let
                         ( sign, signStr ) =
-                            if hour < 0 then
+                            if hour <= 0 then
```

Comment:
This is a bit of a subtle change. Technically, ISO 8601 requires that a 0 offset be represented with a `+` offset separator, while RFC 3339 allows `-`.  (I'm learning more about time formatting standards than I ever wanted to know) I figured it was better to stick to the ISO standard just to make sure it was getting tested, even though it doesn't matter from a parsing standpoint.